task/power/src/main.rs

```diff
@@ -349,6 +359,45 @@ macro_rules! max5970_controller {
     };
 }
 
+fn trace_max5970(dev: &Max5970, sensor: SensorId) {
+    if let Ok(Volts(volts)) = dev.max_vout() {
+        use drv_i2c_devices::max5970::Register;
+
+        if volts < 12.0 {
```

Comment:
What's the thought here?  It seems like we'd still want to log even if the max VOUT is < 12V, since we could be at 11.97V, or something _interesting_ could be happening

Reply:
I wanted to avoid the 3V3 rails on U8 on the Sharkfin. Will add a comment to that effect...